pkgs/tools/security/hash-identifier/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitLab, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "hash-identifier";
+  version = "1.2";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/hash-identifier";
+    rev = "kali/${version}+git20180314-0kali1";
+    sha256 = "1amz48ijwjjkccg6gmdn3ffnyp2p52ksagy4m9gy8l2v5wj3j32h";
```

Comment:
Would using the upstream repo https://github.com/blackploit/hash-identifier be possible?

Reply:
Sure.

Reply:
Well actually, thinking about it more, the main repo doesn't have releases unlike the kali ones. That means this version would have to be "unstable-2021-06-22" or whatever.

Reply:
I see this issue: https://github.com/blackploit/hash-identifier/issues/4 hopefully we'll get a response.

Reply:
Imo when we get a response we can change it, but until then this is fine to merge in its current state.